modules/mockserver/src/test/java/org/testcontainers/containers/MockServerContainerTest.java

```diff
@@ -6,30 +6,55 @@
 import org.mockserver.client.MockServerClient;
 
 import java.io.BufferedReader;
+import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.URL;
 import java.net.URLConnection;
 
+import static org.hamcrest.CoreMatchers.containsString;
 import static org.mockserver.model.HttpRequest.request;
 import static org.mockserver.model.HttpResponse.response;
-import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertThat;
 
 public class MockServerContainerTest {
 
     @ClassRule
-    public static MockServerContainer mockServer = new MockServerContainer();
+    public static MockServerContainer mockServer = new MockServerContainer(MockServerClient.class.getPackage().getImplementationVersion());
+
+    @ClassRule
+    public static MockServerContainer mockServerDefault = new MockServerContainer();
```

Comment:
this will start 2 containers for each test, although only one is actually used. Please consider using `try-with-resources` in the test's body

Reply:
even if its marked as class rule?

Reply:
Of course. Or, maybe, to re-phrase:
If you run a single test, e.g. `default` scenario, it will start both containers, which is not desirable